about.html

```diff
@@ -0,0 +1,182 @@
+                                Committe's judging process, the logo from the Naresh Agrawal won the prize.
+                                After consultations with Mr. Agrawal we optimised the logo for each and every Conference
+                                we shall have
+                            </p>
+                            <div class="col-md-8 px-5 border border-3 mx-auto">
+                                <img class="img-fluid" src="./assests/img/devcon5.webp" alt="devcon logo" />
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <hr class="border border-1" />
+        <section class="speakers d-flex flex-column align-items-center py-4">
+            <div class="container">
+                <div class="title-2 pt-4 d-flex flex-column align-items-center text-center">
+                    <h2 class="m-0 fw-bold">See the past Devcon and I/O events</h2>
+                    <hr />
+                </div>
+                <div class="speaker-wrapper pt-3 pb-4">
+                    <div class="row">
+                        <div class="col-lg-12 text-center text-secondary">
+                            <p>
+                                Take a look at the last two Devcon events which took place in Apple park
+                            </p>
+                            <div class="past-events d-flex w-100 flex-column px-md-5 flex-md-row text-light">
+                                <div class="event-1 mx-md-3 d-flex align-items-center justify-content-center">
+                                    <div>
+                                        <span>2019</span>
+                                        <p>Dev con 2019 in Paris</p>
+                                    </div>
+                                </div>
+                                <div class="event-2 d-flex align-items-center justify-content-center">
+                                    <div>
+                                        <span>2018</span>
+                                        <p>Dev con 2018 in Paris</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer class="bg-dark text-light">
+        <div class="container">
+            <div class="d-flex align-items-center justify-content-between py-5">
+                <div class="ps-3">
+                    <img src="./assests/img/devcon5.webp" width="100" alt="DEV CONN" />
+                </div>
+                <div class="px-3">
+                    2023 Devlabs Developer Conference.<br />
+                    All Rights reserved
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] Your footer is nice on the desktop view, but on the mobile view it is missing the partner's section above it and the footer color has a white background with dark text and logo on the template. Kindly make these changes to make your footer similar to the template.
 - Your design:
 - 
<img width="377" alt="image" src="https://user-images.githubusercontent.com/61919617/217871986-7fbe4c10-5de4-47b7-91d6-e13543ca394a.png">

 - Template:
 - 
<img width="401" alt="image" src="https://user-images.githubusercontent.com/61919617/217872042-efad2362-982a-42a3-9082-68064bb5ecad.png">